spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -137,17 +137,32 @@ def _assigns
           expect(view_spec.received.first).to eq([{:template => "widgets/new"},{}, nil])
         end
 
-        it "converts the filename components into render options" do
-          allow(view_spec).to receive(:_default_file_to_render) { "widgets/new.en.html.erb" }
-          view_spec.render
-
-          if ::Rails::VERSION::STRING >= '3.2'
+        if ::Rails::VERSION::STRING >= '3.2'
+          it "converts the filename components into render options" do
+            allow(view_spec).to receive(:_default_file_to_render) { "widgets/new.en.html.erb" }
+            view_spec.render
             expect(view_spec.received.first).to eq([{:template => "widgets/new", :locales=>['en'], :formats=>[:html], :handlers=>['erb']}, {}, nil])
-          else
+          end
+
+          it "converts the filename with variant into render options" do
+            allow(view_spec).to receive(:_default_file_to_render) { "widgets/new.en.html+fancy.erb" }
+            view_spec.render
+            expect(view_spec.received.first).to eq([{:template => "widgets/new", :locales=>['en'], :formats=>[:html], :handlers=>['erb'], variants: ['fancy']}, {}, nil])
+          end
+
+          it "converts the filename without format into render options" do
+            allow(view_spec).to receive(:_default_file_to_render) { "widgets/new.en.erb" }
+            view_spec.render
+            expect(view_spec.received.first).to eq([{:template => "widgets/new", :locales=>['en'], :handlers=>['erb']}, {}, nil])
+          end
+        else
+          it "uses the filename as a template" do
+            allow(view_spec).to receive(:_default_file_to_render) { "widgets/new.en.html.erb" }
+            view_spec.render
             expect(view_spec.received.first).to eq([{:template => "widgets/new.en.html.erb"}, {}, nil])
           end
         end
-      end
+    end
```

Comment:
This is mis indented isnt it?

Reply:
Done in https://github.com/rspec/rspec-rails/pull/2071/commits/86db9eb72fd42a24db3664cd5c4735b84dcfa59d